www/js/services.js

```diff
@@ -215,78 +117,4 @@ angular.module('emission.services', ['emission.plugin.logger'])
       return window.cordova.plugins.BEMConnectionSettings.getSettings();
     };
 
-})
-
-.factory('Chats', function() {
-  // Might use a resource here that returns a JSON array
-
-  // Some fake testing data
-  var chats = [{
-    id: 0,
-    name: 'Ben Sparrow',
-    lastText: 'You on your way?',
-    face: 'img/ben.png'
-  }, {
-    id: 1,
-    name: 'Max Lynx',
-    lastText: 'Hey, it\'s me',
-    face: 'img/max.png'
-  }, {
-    id: 2,
-    name: 'Adam Bradleyson',
-    lastText: 'I should buy a boat',
-    face: 'img/adam.jpg'
-  }, {
-    id: 3,
-    name: 'Perry Governor',
-    lastText: 'Look at my mukluks!',
-    face: 'img/perry.png'
-  }, {
-    id: 4,
-    name: 'Mike Harrington',
-    lastText: 'This is wicked good ice cream.',
-    face: 'img/mike.png'
-  }, {
-    id: 5,
-    name: 'Ben Sparrow',
-    lastText: 'You on your way again?',
-    face: 'img/ben.png'
-  }, {
-    id: 6,
-    name: 'Max Lynx',
-    lastText: 'Hey, it\'s me again',
-    face: 'img/max.png'
-  }, {
-    id: 7,
-    name: 'Adam Bradleyson',
-    lastText: 'I should buy a boat again',
-    face: 'img/adam.jpg'
-  }, {
-    id: 8,
-    name: 'Perry Governor',
-    lastText: 'Look at my mukluks again!',
-    face: 'img/perry.png'
-  }, {
-    id: 9,
-    name: 'Mike Harrington',
-    lastText: 'This is wicked good ice cream again.',
-    face: 'img/mike.png'
-  }];
```

Comment:
There are a bunch of unused images in `www/img` that might be appropriate to also remove at this point in time

Reply:
Not just the 'chats' images, but also remnants of Habitica, calorie-counting, and some random gifs.

I think the only image still needed is `enketo_bare_150x56.png`.